pkg/kor/all.go

```diff
@@ -254,6 +263,12 @@ func GetUnusedAll(filterOpts *filters.Options, clientset kubernetes.Interface, a
 	outputBuffer.WriteString("\n")
 	noNamespaceResourceMap[clusterRoleDiff.resourceType] = clusterRoleDiff.diff
 
+	storageClassDiff := getUnusedStorageClasses(clientset, filterOpts)
```

Comment:
Please wrap all the non namespaced resources into a separate function.

Reply:
Seperated to 2 new functions `GetUnusedAllNamespaced` & `GetUnusedAllNonNamespaced`, and called both from `GetUnusedAll`.